src/main/java/seedu/address/model/flag/Flag.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.flag;
+
+import static seedu.address.logic.parser.CliSyntax.MISSION_DEADLINE;
+import static seedu.address.logic.parser.CliSyntax.QUEST_DEADLINE;
+
+import java.util.Set;
+
+import com.google.common.collect.ImmutableSet;
+
+public class Flag {
```

Comment:
Maybe we could add a javadoc comment as follows:
Represents the string consisting of a dash followed by an alphabet such as "-m" included in the user's command.